cardano-cli/cardano-cli.cabal

```diff
@@ -200,7 +200,7 @@ library
                       , binary
                       , bytestring
                       , canonical-json
-                      , cardano-api ^>= 8.22
```

Comment:
TODO:
- [ ] use 8.24 after release
- [ ] bump flake.lock
- [ ] bump chap index in cabal.project